sdk/rmscrypto_sdk/CryptoAPI/CryptoAPI.pro

```diff
@@ -23,7 +23,7 @@ CONFIG(debug, debug|release) {
 }
 
 win32:LIBS += -L$$REPO_ROOT/third_party/lib/eay/ -lssleay32 -llibeay32 -lGdi32 -lUser32 -lAdvapi32
-win32:LIBS += -llegacy_stdio_definitions
+#win32:LIBS += -llegacy_stdio_definitions
```

Comment:
Don't we need 'llegacy_stdio_definitions' anymore? If not, please delete it

Reply:
A bit of history: when we first moved to VC 2015, we had to include this lib because of some breaking changes in the runtime from VC 2013. I do not understand why those failures are gone now. Will leave this commented out for now so that it is easier to fix in case this (lack of) issue turns out to be due to a temporary condition on the build machine.